tools/trace_example.txt

```diff
@@ -164,6 +174,12 @@ PID    TID    COMM         FUNC             -
 777    785    automount    SyS_nanosleep    sleep for 500000000 ns
 ^C
 
+The tool automatically adds an `#include` for a few common types, but
+generally you should use the -I argument to have the requisite headers
+files included by the code generator, for compilation to succeed. Use
+either the filename relative to /usr/include, or an absolute path to
+the header file.
```

Comment:
I disagree. The one example here showing struct support "coincidentally" chose one of only 4 or so types that bcc has special support for. This kind of (partial) magic makes exposition easy, then creates frustration when the tool violates the user's expectations. 
So, I strongly feel some explanation like this belongs here, but I'll pare it down to just "Use -I to include the header file" as its shorter. Better to provide the general case that always works, than the convenient case that usually won't.